src/_internal_test_exports/fuzz.rs

```diff
@@ -4,7 +4,8 @@ use std::time::Duration;
 use std::time::Instant;
 
 use crate::change::{SdpAnswer, SdpOffer};
-use crate::dtls::{KeyingMaterial, SrtpProfile};
+use crate::crypto::KeyingMaterial;
```

Comment:
merge imports like the others?

Reply:
Haven't you seen my other PR where I threaten to make all imports like this? :)

Tbh, we are not having consistent imports at this point. I guess I'm ruined by Java, but I hate with a passion imports that break the use over several lines.

```
use foo:{
  blah blah blah
  blah blah blah
}
```

I can live with some being individual and some not though...